amphimixis/shell/local_shell_handler.py

```diff
@@ -55,5 +58,28 @@ def stderr_readline(self) -> str:
             raise BrokenPipeError("Can't read from process' stderr")
         return self.shell.stderr.readline().decode()
 
-    def copy_to_remote(self, source: str, destination: str) -> None:
-        subprocess.check_call(["cp", "-a", source, destination])
+    def copy_to_remote(self, source: str, destination: str) -> int:
+        logging.info("Copying files")
+
+        error_code = subprocess.call(
+            [
+                "rsync",
+                "--checksum",
+                "--archive",
+                "--recursive",
+                "--mkpath",
+                "--copy-links",
+                "--hard-links",
+                "--compress",
+                "--log-file=./amphimixis.log",
+                source,
+                destination,
+            ]
+        )
+
+        if error_code != 0:
+            logging.error("Sources copying error")
+            return -1
+
+        logging.info("Successful copied")
+        return 0
```

Comment:
```suggestion
    def copy_to_remote(self, source: str, destination: str) -> bool:
        logging.info("Copying files")

        error_code = subprocess.call(
            [
                "rsync",
                "--checksum",
                "--archive",
                "--recursive",
                "--mkpath",
                "--copy-links",
                "--hard-links",
                "--compress",
                "--log-file=./amphimixis.log",
                source,
                destination,
            ]
        )

        if error_code != 0:
            logging.error("Sources copying error")
            return False

        logging.info("Successful copied")
        return True
```

it's not C. Python has wonderful boolean type